spread.yaml

```diff
@@ -61,7 +61,7 @@ backends:
                 kernel: GRUB 2
                 workers: 4
             - fedora-25-64:
-                workers: 1
+                workers: 2
```

Comment:
Yes, let's please keep it down to 1 while we have so many disabled tests. We should only increase it further once this becomes the bottleneck (IOW, it finishes last).